src/lib/encoder.ts

```diff
@@ -143,6 +143,123 @@ const decodeCompactBytesArray = (compactBytesArray: string): string[] => {
   return encodedValues;
 };
 
+const encodeBytesNCompactBytesArray = (
+  values: string[],
+  numberOfBytes: number,
+): string => {
+  for (let i = 0; i < values.length; i++) {
```

Comment:
cant you do `values.forEach()`?

Reply:
good catch, I made all the functions more JS like